packages/ts/file-router/src/types.d.ts

```diff
@@ -25,6 +25,12 @@ export type ViewConfig = Readonly<{
    */
   route?: string;
 
+  /**
+   * Allows giving marking view as using server side parent layout
```

Comment:
```suggestion
   * Set to true to indicate that the view as using server side parent layout
```

Reply:
That is already possible, but the linking is done on the server so it is not needed as information on the client.